apps/studio/src/lib/data/tools.ts

```diff
@@ -62,13 +62,16 @@ export const Mutators = {
 
   /**
    * Stringify bit data for use in json / UIs
+   * @param {DBConnection|null} connection: The connection object used within Vue
    * @param  {any} value
    * @returns JsonFriendly
    */
-  bitMutator(value: any): JsonFriendly {
-    // SQL Server bit data type is an integer data type
-    if (this.connection?.connectionType === "sqlserver") return value;
-
+  bitMutator(connection: DBConnection|null, value: any): JsonFriendly {
```

Comment:
Ok, couple more changes here. Sorry for late comments.

1. This doesn't need the full connection, it should just take the `dialect` which is typed and can be matched. It should default to `undefined` using the question mark. See dialects/index.ts :
```ts
bit1Mutator(value: any, dialect?: Dialect): JsonFriendly {
 if (dialect && dialect === 'sqlserver') return value

|

```
2. Also because this is optional, it should be the last argument.

Reply:
dialect will need to be the first argument in there because it is passed in as the argument when setting up the `bind` function in `resolveDataMutator`